s3recovery/s3recovery/s3recovercorruption.py

```diff
@@ -53,8 +53,13 @@ def cleanup_bucket_metadata_entries(self, index_id):
         data_as_dict = self.parse_index_list_response(list_index_response)
         key_list = list(data_as_dict.keys())
         for key in key_list:
-            key_part = key.split('/')[1]
-            if key_part not in self.common_keys:
+            if (key.count('/') == 1):
```

Comment:
use a constant for '/' like METADATA-INDEX-KEYNAME-SEPARATOR.

Reply:
Done